lib/common/virus_scan.rb

```diff
@@ -7,8 +7,7 @@ module VirusScan
     def scan(file_path)
       # `clamd` runs within service group, needs group read
       File.chmod(0o640, file_path)
-      args = ['--config', Rails.root.join('config', 'clamd.conf').to_s]
-      ClamScan::Client.scan(location: file_path, custom_args: args)
+      ClamScan::Client.scan(location: file_path)
```

Comment:
Moved the custom config to the Dockerfile, because that's where it truly belongs.